drivers/media/i2c/imx477.c

```diff
@@ -983,8 +983,8 @@ static const struct imx477_mode supported_modes_10bit[] = {
 			 * rectangle once the driver is expanded to represent
 			 * its processing blocks with multiple subdevs.
 			 */
-			.left = 4,
-			.top = 0,
+			.left = 12,
```

Comment:
I wonder if generally ```IMX477_PIXEL_ARRAY_LEFT + 4``` is more sensible for these values to make it more explicit.
Probably a more general question than worrying over fixing here.

Reply:
Agree, I did want to do it as `IMX477_PIXEL_ARRAY_LEFT + 4`, but I though it best that I match the changes in imx219.

Reply:
The existing isn't always right :-)

If a sensor was added that only had a cropped mode then you wouldn't know if the offset were correct or not. Making it ```_PIXEL_ARRAY_LEFT + X``` makes it obvious that it has been considered.

Reply:
Ok, will post an update.